jerry-core/parser/js/js-parser.c

```diff
@@ -1951,16 +1961,21 @@ parser_post_processing (parser_context_t *context_p) /**< context */
 
     while (flags & (CBC_HAS_LITERAL_ARG | CBC_HAS_LITERAL_ARG2))
     {
-      uint8_t first_byte = page_p->bytes[offset];
+      uint16_t first_byte = page_p->bytes[offset];
 
       uint8_t *opcode_pos_p = dst_p - 1;
-      *dst_p++ = first_byte;
+      *dst_p++ = (uint8_t) first_byte;
```

Comment:
Is it ok to cast from `uint16_t` to `uint8_t` here?

Reply:
It's a safe cast since `first_byte` comes from `page_p->bytes[offset]` which contains `uint8_t`s.